### ISSUE TITLE
Image-only PDFs are not being re-scanned as images

### ISSUE THREAD

**Issue by poveden** [axa-group/Parsr#132]
**Summary**

PDF files that only contain images (or PDF files where the extractor yields no text elements) are not converted to images and re-scanned, due to a bug in `server/bin/index.ts`:

```
TypeError: Reduce of empty array with no initial value
    at Array.reduce (<anonymous>)
    at /opt/app-root/src/dist/bin/index.js:104:85
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

https://github.com/axa-group/Parsr/blob/7d4127ab16a61a3f43d69400014d3c93fe44664b/server/bin/index.ts#L120

This bug should be easily fixed by adding `0` as the initial value in the `reduce()` call:

```js
const nbTexts = doc.pages.map(p => p.elements.length).reduce((a, b) => a + b, 0);
```

**Steps To Reproduce**

1. Post to the /api/document endpoint a PDF file that contains only images, along with the [example configuration file](https://github.com/axa-group/Parsr/blob/develop/docs/configuration-file.md#5-exempli-gratia)
3. Examine the logs

**Expected behavior**

The PDF file scan results should yield expected values.

**Actual behavior**

Besides the error message detailed above, a 404 error is received when trying to obtain the results from `/api/json/{id}`, `/api/text/{id}` or `/api/markdown/{id}`.

**Environment**

- Docker Desktop for Windows 2.1.0.3 (38240), engine 19.03.2
- Docker image `axarev/parsr`, ID `74a4e7320413`

**Comment by aarohijohal** [comment]
I'm not seeing the same behaviour under a bare metal installation on my MacOS 10.4 running node v12.

**Comment by marianorodriguez** [comment]
I will close this, as it was resolved in commit d5bed5d and  is already merged into develop
